# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\Users\lockon\workspace\version-2_12-dev_test\data\module\Net\UserAgent
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Mobile.php
--- Mobile.php ベース (BASE)
+++ Mobile.php ローカルでの変更 (LOCAL がベース)
@@ -424,7 +424,8 @@
             'blackberry9800', // Torch
             'webOS',          // Palm Pre Experimental
             'incognito',      // Other iPhone browser
-            'webmate'         // Other iPhone browser
+            'webmate',         // Other iPhone browser
+            'Windows Phone OS' // Windows Phone
         );
 
         $pattern = implode("|", $useragents);
